### DISCUSSION TITLE
Breaking changes between 3.0 & 3.1 ?

### DISCUSSION THREAD

**Discussion by tlosbar** [saleor/saleor#8667] [Q&A]
Hello,

There are some breaking changes from 3.0 to 3.1? 

I'm currently developing an e-commerce website based on the 3.0 version as it was recommended to start news projects with, instead of 2.11 (and I love it!). But I see, there is some feature like _Anonymous checkout without email_  that will be merged into 3.1. 
Is this version ready for production?

Thanks

**Comment by karolkielecki** [comment]
@tlosbar Hi! Thanks for asking. Yes, a few new features will be released in tandem with 3.1, but it's still an alpha stage. We do not recommend it for production. We recommend using v3.0 (the latest beta). We'll provide more info about 3.1 as soon as we can.
